code/game/objects/items/devices/scanners.dm

```diff
@@ -119,6 +119,19 @@ GENE SCANNER
 	user.visible_message("<span class='notice'>[user] analyzes [M]'s vitals.</span>", \
 						"<span class='notice'>You analyze [M]'s vitals.</span>")
 
+	//monkestation edit start
+	if(user.eye_blind) //based on the examinate verb in mob/mob.dm
```

Comment:
Radioactive scanners (Traitor item) instantly irradiates even if interrupted.
Not a big deal, but exists